### PR TITLE
Add cli config flag mergin possibility

### PR DESCRIPTION
**Pull request by regisphilibert**
to
* CLI
* Getting Starter/Configure Hugo

### CODE REVIEW

File: content/commands/hugo.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ hugo [flags]
       --cacheDir string            filesystem path to cache directory. Defaults: $TMPDIR/hugo_cache/
       --canonifyURLs               if true, all relative URLs will be canonicalized using baseURL
       --cleanDestinationDir        remove files from destination not found in static directories
-      --config string              config file (default is path/config.yaml|json|toml)
+      --config string              config file(s), multiple files (a.toml,b.toml.c.toml) will be merged (default is path/config.yaml|json|toml)
```
Comment (bep) [inline content/commands/hugo.md]: The content below /commands is autogenerated from code and this will be overwritten eventually.

Comment (kaushalmodi) [reply content/commands/hugo.md]: Might be better to have:

```markdown
### Options
<!-- Do NOT edit this section manually, it is auto-generated by ?? -->
```

*(where ?? be replaced with whatever is auto-generating this part.)*

Comment (regisphilibert) [reply content/commands/hugo.md]: I see. 
If it's not about to be overwritten tomorrow, we could still add it and have you or me (depending on the complexity) modify the sentence directly to the hugo command --help function ?
I was about to look into adding the sentence to the cli --help anyway.

Comment (bep) [reply content/commands/hugo.md]: https://github.com/gohugoio/hugo/blob/master/commands/hugo.go#L229

**Comment by regisphilibert** [comment]
here you go (not sure you've been notified as I can't request you as a reviewer)
https://github.com/gohugoio/hugo/pull/4370

**Comment by kaushalmodi** [comment]
I could be wrong, but I thought this change needed to then happen in hugo core?

**Comment by regisphilibert** [comment]
yes the hugo core related PR is mentioned above. (https://github.com/gohugoio/hugo/pull/4370), maybe this hugoDocs issue can then be closed.

**Comment by bep** [comment]
Sorry for the delay, but I think we can close this.

**Pull request closed without merging**